lib/instrumentation/modules/mongodb.js

```diff
@@ -8,18 +8,40 @@ const HOSTNAME_PORT_RE = /^([^:]+):(\d+)$/
 
 module.exports = (mongodb, agent, { version, enabled }) => {
   if (!enabled) return mongodb
-  if (!semver.satisfies(version, '>=3.3')) {
+  if (!semver.satisfies(version, '>=3.3 < 5.0')) {
```

Comment:
nit:

```suggestion
  if (!semver.satisfies(version, '>=3.3 <5.0')) {
```

If you bother to take this suggestion, don't bother waiting for a test re-run to pass on my account.